### DISCUSSION TITLE
Static File serving

### DISCUSSION THREAD

**Discussion by SEED-Foundation** [serverpod/serverpod#1861] [Q&A] [answered]
I did'nt find anything in the doc about that , how to serve static file for example if i ceate a folder call images then i have multiple image inside it how to serve them ?

**Comment by hientrung** [comment, marked as answer]
in server.dart
```dart
  // Serve all files in the /static directory.
  pod.webServer.addRoute(
    RouteStaticDirectory(serverDirectory: 'static', basePath: '/'),
    '/*',
  );
```

**Reply by vlidholt** [reply]
Just to add to this answer, this is a feature of Relic, the web server of Serverpod. Relic is under active development. A bit more info is here:
https://docs.serverpod.dev/concepts/webserver
